e2e/cypress/e2e/translations/mtSettings.cy.ts

```diff
@@ -24,7 +24,7 @@ describe('Machine translation settings', () => {
     }
   });
 
-  it('will update default settings', () => {
+  it('will update default settings', { retries: 2 }, () => {
```

Comment:
Is this the retries 2 intended? If it's really fleaky and it's unrsolvable, I would allow 5 retrials, otherwise, let's remove it.

Reply:
It is the issue with sockets because I'm checking the auto translation is set correctly. I can set it to 5 if you like